src/api/seeders/20190621184055-user.js

```diff
@@ -9,7 +16,280 @@ module.exports = {
+          email: 'alain2@gmail.com',
+          password: hashedPassword,
+          createdAt: new Date(),
+          updatedAt: new Date()
+        },
+        {
+          username: 'BurindiAlain3',
+          email: 'alain3@gmail.com',
+          password: hashedPassword,
+          createdAt: new Date(),
+          updatedAt: new Date()
+        },
+        {
+          username: 'BurindiAlain44',
+          email: 'alain44@gmail.com',
+          password: hashedPassword,
+          createdAt: new Date(),
+          updatedAt: new Date()
+        },
+        {
+          username: 'BurindiAlain5',
+          email: 'alain5@gmail.com',
+          password: hashedPassword,
+          createdAt: new Date(),
+          updatedAt: new Date()
+        },
+        {
+          username: 'BurindiAlain6',
+          email: 'alain6@gmail.com',
+          password: hashedPassword,
+          createdAt: new Date(),
+          updatedAt: new Date()
+        },
+        {
+          username: 'BurindiAlain7',
+          email: 'alain7@gmail.com',
+          password: hashedPassword,
+          createdAt: new Date(),
+          updatedAt: new Date()
+        },
+        {
+          username: 'BurindiAlain8',
+          email: 'alain8@gmail.com',
+          password: hashedPassword,
+          createdAt: new Date(),
+          updatedAt: new Date()
+        },
+        {
+          username: 'BurindiAlain9',
+          email: 'alain9@gmail.com',
+          password: hashedPassword,
+          createdAt: new Date(),
+          updatedAt: new Date()
+        },
+        {
+          username: 'BurindiAlain10',
+          email: 'alain10@gmail.com',
+          password: hashedPassword,
+          createdAt: new Date(),
+          updatedAt: new Date()
+        },
+        {
+          username: 'BurindiAlain11',
+          email: 'alain11@gmail.com',
+          password: hashedPassword,
+          createdAt: new Date(),
+          updatedAt: new Date()
+        },
+        {
+          username: 'BurindiAlain12',
+          email: 'alain12@gmail.com',
+          password: hashedPassword,
+          createdAt: new Date(),
+          updatedAt: new Date()
+        },
+        {
+          username: 'BurindiAlain13',
+          email: 'alain13@gmail.com',
+          password: hashedPassword,
+          createdAt: new Date(),
+          updatedAt: new Date()
+        },
+        {
+          username: 'BurindiAlain14',
+          email: 'alain14@gmail.com',
+          password: hashedPassword,
+          createdAt: new Date(),
+          updatedAt: new Date()
+        },
+        {
+          username: 'BurindiAlain15',
+          email: 'alain15@gmail.com',
+          password: hashedPassword,
+          createdAt: new Date(),
+          updatedAt: new Date()
+        },
+        {
+          username: 'BurindiAlain16',
+          email: 'alain16@gmail.com',
+          password: hashedPassword,
+          createdAt: new Date(),
+          updatedAt: new Date()
```

Comment:
why all these users??

Reply:
I needed to mock the idea of pagination.